{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# OpenMS Glossary\n",
    "\n",
    "A glossary of common terms used throughout OpenMS documentation.\n",
    "\n",
    "LC-MS  \n",
    "Liquid Chromatography-Mass Spectrometry.\n",
    "\n",
    "Liquid chromatography  \n",
    "An analytical technique used to separate molecules of interest.\n",
    "\n",
    "mass spectrometry  \n",
    "An analytical technique used to identify and quantify molecules of\n",
    "interest.\n",
    "\n",
    "peptides  \n",
    "A short chain of amino acids.\n",
    "\n",
    "FASTA format  \n",
    "A text-based format for representing nucleotide or amino acid sequences.\n",
    "\n",
    "octadecyl (C18)  \n",
    "An alkyl radical C(18)H(37) derived from an octadecane by removal of one\n",
    "hydrogen atom.\n",
    "\n",
    "Mass  \n",
    "Mass is a measure of the amount of matter that an object contains. In\n",
    "comparison to often used term weight, which is a measure of the force of\n",
    "gravity on that object.\n",
    "\n",
    "ion  \n",
    "Any `atom` or group of atoms that bears one or more positive or negative\n",
    "electrical charges. Positively charged are cations, negavtively charged\n",
    "anions.\n",
    "\n",
    "electrospray ionization (ESI)  \n",
    "A technique used in mass spectrometry to produce ions.\n",
    "\n",
    "MS^2  \n",
    "See `MS/MS`.\n",
    "\n",
    "atom  \n",
    "An atom is the smallest unit of ordinary matter that forms a chemical\n",
    "element.\n",
    "\n",
    "electrospray ionization  \n",
    "A technique used in mass spectrometry to produce ions using an\n",
    "electrospray in which a high voltage is applied to a liquid to create an\n",
    "`aerosol`.\n",
    "\n",
    "aerosol  \n",
    "An aerosol is a suspension of fine solid particles or liquid droplets in\n",
    "air or another gas.\n",
    "\n",
    "time-of-flight (TOF)  \n",
    "A measurement of the time taken by an object, particle of wave (be it\n",
    "acoustic, electromagnetic, e.t.c) to travel a distance through a medium.\n",
    "\n",
    "quadrupole mass filters  \n",
    "A mass filter allowing one mass channel at a time to reach the detector\n",
    "as the mass range is scanned.\n",
    "\n",
    "orbitrap analyzers  \n",
    "In mass spectrometry, an ion trap mass analyzer consisting of an outer\n",
    "barrel-like electrode and a coaxial inner spindle-like electrode that\n",
    "traps ions in an orbital motion around the spindle. A high resoltion\n",
    "mass spectrometry analyzer.\n",
    "\n",
    "MS(1)  \n",
    "First stage to get a spectra. A sample is injected into the mass\n",
    "spectrometer, ionized, accelerated and analyzed by mass spectrometry.\n",
    "\n",
    "MS(2)  \n",
    "Ions from MS1 spectra are then selectively fragmented and analyzed by a\n",
    "second stage of mass spectrometry (MS2) to generate the spectra for the\n",
    "ion fragments.\n",
    "\n",
    "MS/MS  \n",
    "Tandem mass spectrometry, MS^2^, a technique where two or more mass\n",
    "analyzers are coupled together using an additional reaction step to\n",
    "increase their abilities to analyse chemical samples.\n",
    "\n",
    "collision-induced dissociation (CID)  \n",
    "A mass spectrometry technique to induce fragmentation of selected ions\n",
    "in the gas phase. Also known as Collision induced dissociation.\n",
    "\n",
    "TOPP  \n",
    "The OpenMS Pipeline.\n",
    "\n",
    "MSGFPlusAdapter  \n",
    "Adapter for the MS-GF+ protein identification (database search) engine.\n",
    "More information is available in the [OpenMS API reference\n",
    "documentation](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_MSGFPlusAdapter.html).\n",
    "\n",
    "LuciphorAdapter  \n",
    "Adapter for the LuciPHOr2: a site localisation tool of generic\n",
    "post-translational modifications from tandem mass spectrometry data.\n",
    "More information is available in the [OpenMS API reference\n",
    "documentation](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_LuciphorAdapter.html).\n",
    "\n",
    "pyOpenMS  \n",
    "pyOpenMS is an open-source Python library for mass spectrometry,\n",
    "specifically for the analysis of proteomics and metabolomics data in\n",
    "Python.\n",
    "\n",
    "TOPP tools  \n",
    "OpenMS provides a number of functions that process mass spectrometry\n",
    "data called TOPP tools. All TOPP tools are described in the [OpenMS API\n",
    "reference\n",
    "documentation](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_documentation.html).\n",
    "\n",
    "UTILS  \n",
    "Besides `TOPP tools`, OpenMS offers a range of other tools. They are not\n",
    "included in `TOPP` as they are not part of typical analysis pipelines.\n",
    "More information is present in [OpenMS API reference\n",
    "documentation](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/UTILS_documentation.html).\n",
    "\n",
    "TOPPView  \n",
    "TOPPView is a viewer for MS and HPLC-MS data.\n",
    "\n",
    "Nightly Snapshot  \n",
    "Untested installers and containers are known as the nightly snapshot.\n",
    "\n",
    "proteomics  \n",
    "Proteomics is the large-scale study of proteins.\n",
    "\n",
    "proteins  \n",
    "Proteins are vital parts of living organisms, with many functions, for\n",
    "example composing the structural fibers of muscle to the enzymes that\n",
    "catalyze the digestion of food to synthesizing and replicating DNA.\n",
    "\n",
    "MascotAdapter  \n",
    "Used to identifies peptides in MS/MS spectra. Read more about\n",
    "MascotAdapter in the [OpenMS API reference\n",
    "documentation](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/TOPP_MascotAdapter.html).\n",
    "\n",
    "HPLC-MS  \n",
    "Data produced by High performance liquid chromatography (HPLC) separates\n",
    "components of a mixture, whereas mass spectrometry (MS) offers the\n",
    "detection tools to identify them.\n",
    "\n",
    "mzML  \n",
    "The mzML format is an open, XML-based format for mass spectrometer\n",
    "output files, developed with the full participation of vendors and\n",
    "researchers in order to create a single open format that would be\n",
    "supported by all software.\n",
    "\n",
    "mzData  \n",
    "mzData was the first attempt by the Proteomics Standards Initiative\n",
    "(PSI) from the Human Proteome Organization (HUPO) to create a\n",
    "standardized format for Mass Spectrometry data. This format is now\n",
    "deprecated, and replaced by mzML.\n",
    "\n",
    "mzXML  \n",
    "mzXML is an open data format for storage and exchange of mass\n",
    "spectroscopy data, developed at the SPC/Institute for Systems Biology.\n",
    "\n",
    "spectra  \n",
    "Plural of spectrum.\n",
    "\n",
    "mass spectrum  \n",
    "A mass spectrum is a plot of the ion signal as a function of the\n",
    "mass-to-charge ratio. A mass spectrum is produced by a single mass\n",
    "spectrometry run. These spectra are used to determine the elemental or\n",
    "isotopic signature of a sample, the masses of particles and of\n",
    "molecules, and to elucidate the chemical identity or structure of\n",
    "molecules and other chemical compounds. OpenMS represents a one\n",
    "dimensional mass spectrum using the class\n",
    "[MSSpectrum](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1MSSpectrum.html).\n",
    "\n",
    "m/z  \n",
    "mass to charge ratio.\n",
    "\n",
    "retention time  \n",
    "retention time (RT) in liquid chromatography, is the time it takes for a\n",
    "separated analyte to move through the stationary phase.\n",
    "\n",
    "ProteoWizard  \n",
    "ProteoWizard is a set of open-source, cross-platform tools and libraries\n",
    "for proteomics data analyses. It provides a framework for unified mass\n",
    "spectrometry data file access and performs standard chemistry and LCMS\n",
    "dataset computations.\n",
    "\n",
    "PepNovo  \n",
    "PepNovo is a de novo sequencing algorithm for `MS/MS` `spectra`.\n",
    "\n",
    "de novo peptide sequencing  \n",
    "A peptideâ€™s amino acid sequence is inferred directly from the precursor\n",
    "peptide mass and tandem mass spectrum (`MS/MS` or `MS^3`) fragment ions,\n",
    "without comparison to a reference proteome.\n",
    "\n",
    "TOPPAS  \n",
    "An assistant for GUI-driven TOPP workflow design. It is recommended to\n",
    "use OpenMS through the KNIME plugins.\n",
    "\n",
    "chromatogram  \n",
    "A two-dimensional plot that describes the amount of analyte eluted from\n",
    "a chromatography versus the analyte's retention time. OpenMS represents\n",
    "a chromatogram using the class\n",
    "[MSChromatogram](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/structOpenMS_1_1Interfaces_1_1Chromatogram.html).\n",
    "\n",
    "KNIME  \n",
    "An advanced workflow editor which OpenMS provides a plugin for.\n",
    "\n",
    "SILAC  \n",
    "Stands for Stable isotope labeling using amino acids in cell culture.\n",
    "\n",
    "iTRAQ  \n",
    "Stands for Isobaric tags for relative and absolute quantitation.\n",
    "\n",
    "TMT  \n",
    "Tandem Mass Tag (TMT) is a mass spectrometry based system designed to\n",
    "identify and quantify proteins in different samples.\n",
    "\n",
    "SRM  \n",
    "Selected reation monitoring is a mass spectrometry technique for small\n",
    "molecule analysis.\n",
    "\n",
    "SWATH  \n",
    "Stands for Sequential acquisition of all theoretical fragment ion\n",
    "spectra.\n",
    "\n",
    "OpenMS API  \n",
    "An interface that allows developers to use OpenMS core library classes\n",
    "and methods.\n",
    "\n",
    "RT  \n",
    "Retention time.\n",
    "\n",
    "MS  \n",
    "Mass Spectrometry\n",
    "\n",
    "TOPP tool  \n",
    "OpenMS offers a vast array of TOPP tools for processing, analyzing and\n",
    "visualizing mass spectrometry data.\n",
    "\n",
    "MS^3  \n",
    "Multi-stage Mass Spectrometry\n",
    "\n",
    "feature  \n",
    "An LC-MS feature represents the combined isotopic mass traces of a\n",
    "detected chemical compound. The chromatographic peak shape of a feature\n",
    "is defined by the interaction of the analyte with the LC column. Each\n",
    "feature contains information on retention time, mass-to-charge ratio,\n",
    "intensity and overall quality. OpenMS represents a feature using the\n",
    "class\n",
    "[Feature](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1Feature.html).\n",
    "\n",
    "feature map  \n",
    "A feature map is a collection of features identified in a mass spectrum\n",
    "from a single experiment. One feature map can contain many features.\n",
    "OpenMS represents a feature map using the class\n",
    "[FeatureMap](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1FeatureMap.html).\n",
    "\n",
    "consensus feature  \n",
    "Features from replicate experiments with similar retention times and m/z\n",
    "values are linked and considered a consensus feature. A consensus\n",
    "feature contains information on the common retention time and m/z values\n",
    "as well as intensities for each sample. OpenMS represents a consensus\n",
    "feature using the class\n",
    "[ConsensusFeature](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1ConsensusFeature.html).\n",
    "\n",
    "consensus map  \n",
    "A consensus map is a collection of\n",
    "`consensus features <consensus feature>` identified from mass spectra\n",
    "across replicate experiments. One consensus map can contain many\n",
    "consensus features. OpenMS represents a consensus map using the class\n",
    "[ConsensusMap](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1ConsensusMap.html).\n",
    "\n",
    "peak  \n",
    "A single raw data point in a chromatogram or a mass spectrum. OpenMS\n",
    "represents a peak in a chromatogram using the class\n",
    "[ChromatogramPeak](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1ChromatogramPeak.html).\n",
    "OpenMS represents a single, one-dimensional peak in a mass spectrum\n",
    "using the class\n",
    "[PeakID](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1Peak1D.html).\n",
    "\n",
    "MSExperiment  \n",
    "An OpenMS class used to represent a single mass spectrometry run. [Read\n",
    "the documentation for further\n",
    "information](https://abibuilder.cs.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1MSExperiment.html)."
   ],
   "id": "6e5c9994-e9ca-4219-af49-e8804fcdcefe"
  }
 ],
 "nbformat": 4,
 "nbformat_minor": 5,
 "metadata": {}
}
